marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/algorithms/schemaElements/suite-setup.xqy

```diff
@@ -0,0 +1,22 @@
+xquery version "1.0-ml";
+import module namespace hub-test = "http://marklogic.com/data-hub/test" at "/test/data-hub-test-helper.xqy";
+hub-test:reset-hub()
+;
+import module namespace test = "http://marklogic.com/test" at "/test/test-helper.xqy";
+declare option xdmp:mapping "false";
+
+let $schema := test:get-test-file("schema.xsd")
```

Comment:
Is the schema necessary for this to work? I see that the content document references it, but does that affect how mastering works here?

Reply:
This comment covers it: https://project.marklogic.com/jira/browse/DHFPROD-7099?focusedCommentId=198738&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-198738 

The basic summary is that things broke due to us passing in the node instead of the value into the matching functions. We didn't catch any issues with this approach due to our XML not having a schema but if a schema defines an element as something other than string the implicit casting doesn't work.